### DISCUSSION TITLE
Self-Hosting suddenly can't login - asking for solution

### DISCUSSION THREAD

**Discussion by yanggut** [plausible/analytics#3049] [Self-Hosted Support]
Hi there, 

may I ask for your help: 

We have set up the self-host plausible check for our website. It works for almost 2 years, all good. 

But one days 3 weeks ago, suddenly we couldn't login any more, although both Email and password didn't change. We tried to reset the password follow the steps, but didn't get the Email to reset the password. 

We also tried to set up the account with another Email, same didn't get the Email with the activation code. 

Do you have any idea, how to solve that, or what could the problem? And do you know how can we at least keep the stats from the last 2 years? 

Thank you so very much for your support. 

Have a great time! 

Best  Regards

**Comment by yanggut** [comment]
forgot to mention: we still get the weekly report per Email, so looks like the plausible analytics is still running, just couldn't login to get the details. 

Your feedback, suggestion and solutions are much appreciated.

**Reply by ruslandoga** [reply]
👋 @yanggut 

You can try listing the users to verify they still exists.

Here's how you can do that with the default `docker-compose.yml` setup located in `~/hosting`
```console
$ cd hosting
$ docker compose exec plausible bin/plausible remote
```
this would start an interactive Elixir shell:
```elixir
iex> import Ecto.Query
iex> "users" |> select([u], map(u, [:email, :password_hash])) |> Plausible.Repo.all()
```
and for the last command you'd get output similar to this:
```elixir
[
  %{
    email: "ruslan@yay.cam",
    password_hash: "$2b$12$oiWL23YRjYtDl1duwfDXfe3jd0yF.Sjahk1KbZwAPJOYEFvOrlQWO"
  },
  %{
    email: "dogaruslan@gmail.com",
    password_hash: "$2b$12$0vNmBTmmmMQcjDNeQniEwO.4XA7s6QbsksxBF21dZT/bj2NAU1vkW"
  }
]
```
This would serve as a sanity check the users still exist in the system.

**Reply by ruslandoga** [reply]
There might be something useful in the logs as well:

```console
$ cd hosting
$ docker compose logs plausible
```
Especially around the time when your logins fail.

**Reply by yanggut** [reply]
great, very helpful, thank you so very much for your solution. We have solved our problem.

Have a great time!
